From 61dac172bdb14c5a37713078828ea8c8f78c7eb6 Mon Sep 17 00:00:00 2001
From: Martin Sehnoutka <msehnout@redhat.com>
Date: Thu, 29 Sep 2016 13:53:16 +0200
Subject: [PATCH 18/59] Change the default log file in configuration.

Previous "default" value was wrong.
tunables.c:262 => install_str_setting("/var/log/xferlog",
&tunable_xferlog_file);
---
 RedHat/vsftpd.log | 6 ++++++
 vsftpd.conf       | 2 +-
 2 files changed, 7 insertions(+), 1 deletion(-)

diff --git a/RedHat/vsftpd.log b/RedHat/vsftpd.log
index d338de8..14731c1 100644
--- a/RedHat/vsftpd.log
+++ b/RedHat/vsftpd.log
@@ -3,3 +3,9 @@
     nocompress
     missingok
 }
+
+/var/log/xferlog {
+    # ftpd doesn't handle SIGHUP properly
+    nocompress
+    missingok
+}
diff --git a/vsftpd.conf b/vsftpd.conf
index ae6c6c9..39d1955 100644
--- a/vsftpd.conf
+++ b/vsftpd.conf
@@ -50,7 +50,7 @@ connect_from_port_20=YES
 #
 # You may override where the log file goes if you like. The default is shown
 # below.
-#xferlog_file=/var/log/vsftpd.log
+#xferlog_file=/var/log/xferlog
 #
 # If you want, you can have your log file in standard ftpd xferlog format.
 # Note that the default log file location is /var/log/xferlog in this case.
-- 
2.14.4

